README.md

```diff
@@ -185,26 +191,64 @@ inoremap <silent><expr> <C-e>     compe#close('<C-e>')
 - [snippets.nvim](https://github.com/norcalli/snippets.nvim)
 - [nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter)
 
-
 ## FAQ
 
-#### How to use LSP snippet?
+### How to use LSP snippet?
 
 1. Set `snippetSupport=true` for LSP capabilities.
 
-```lua
-local capabilities = vim.lsp.protocol.make_client_capabilities()
-capabilities.textDocument.completion.completionItem.snippetSupport = true
+   ```lua
+   local capabilities = vim.lsp.protocol.make_client_capabilities()
+   capabilities.textDocument.completion.completionItem.snippetSupport = true
 
-require'lspconfig'.rust_analyzer.setup {
-  capabilities = capabilities,
-}
-```
+   require'lspconfig'.rust_analyzer.setup {
+     capabilities = capabilities,
+   }
+   ```
 
 2. Install `vim-vsnip`
 
-`Plug 'hrsh7th/vim-vsnip'`
+   `Plug 'hrsh7th/vim-vsnip'`
 
+### How to use tab to navigate completion menu?
+
+`Tab` and `S-Tab` keys need to be mapped to `<C-n>` and `<C-p>` when completion
+menu is visible. Following example will use `Tab` and `S-Tab` (shift+tab) to
+navigate completion menu and jump between
+[vim-vsnip](https://github.com/hrsh7th/vim-vsnip) placeholders when possible:
+
+```lua
+local t = function(str)
+  return vim.api.nvim_replace_termcodes(str, true, true, true)
+end
+
+-- Use (s-)tab to:
+--- move to prev/next item in completion menuone
+--- jump to prev/next snippet's placeholder
+_G.tab_complete = function()
+  if vim.fn.pumvisible() == 1 then
+    return t "<C-n>"
+  elseif vim.fn.call("vsnip#available", {1}) == 1 then
+    return t "<Plug>(vsnip-expand-or-jump)"
+  else
+    return t "<Tab>"
+  end
+end
+_G.s_tab_complete = function()
+  if vim.fn.pumvisible() == 1 then
+    return t "<C-p>"
+  elseif vim.fn.call("vsnip#jumpable", {-1}) == 1 then
+    return t "<Plug>(vsnip-jump-prev)"
+  else
+    return t "<S-Tab>"
+  end
+end
+
+map("i", "<Tab>", "v:lua.tab_complete()", {expr = true})
```

Comment:
Please use Vim style mapping (or add `map` function here

Lua style mappings are bit annoying... Hmm...(due to 
 `vim.api.nvim_replace_termcodes`

Reply:
Ah, missed that, sorry. Added full api call. Is that OK or would you prefer to have shortcut like `local map = vim.api.nvim_set_keymap`?

Reply:
Full api is good. Thank you!